src/backend/src/utils/errors.utils.ts

```diff
@@ -38,6 +38,36 @@ export class AccessDeniedException extends HttpException {
   }
 }
 
+export class AccessDeniedAdminException extends AccessDeniedException {
```

Comment:
I think calling this `AccessDeniedAdminOnlyException` would make more sense and be in line with the other ones